app/src/components/MainPage.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <PageHeader :showMenu="showMenu" :userInfo="userInfo" @toggle="toggleMenu" />
```

Comment:
place header in App.vue [here](https://github.com/SolomiiaLomnytska22/solomiia-training/pull/1/files#diff-12547a84bf202040b65812db58c16a324313390180fb3cbb9bed08b1038055a3R3)